nether/portal.lua

```diff
@@ -176,6 +199,62 @@ minetest.register_chatcommand("from_hell", {
 	end
 })
 
+-- Useful for debugging Nether player state tracking. Written by Deathwing777
+minetest.register_chatcommand("in_hell", {
+	params = "[<player_name>]",
+	description = "Is the player in hell?",
+	func = function(name, pname)
+		if not minetest.check_player_privs(name, {nether=true}) then
+			return false,
+				"You need the nether priv to execute this chatcommand."
+		end
+		if not player_exists(pname) then
+			pname = name
+		end
+		local player = minetest.get_player_by_name(pname)
+		if not player then
+			return false, "Something went wrong."
+		end
+		
+		status = pname.." is in the "
```

Comment:
>     portal.lua:219:3: setting non-standard global variable status

`status` should be changed to a local variable.

Reply:
oops. Still getting used to Lua.